pages.es/android/logcat.md

```diff
@@ -0,0 +1,16 @@
+# logcat
+
+> Vuelca un registro de mensajes del sistema, incluidos los seguimientos de pila, los casos de error del sistema y los mensajes que escribes desde tu app con la clase Log.
+> Más información: <https://developer.android.com/studio/command-line/logcat>.
+
+- Mostrar registros del sistema:
```

Comment:
For Romance languages, the present indicative fits better.
```suggestion
- Muestra registros del sistema:
```

Reply:
Thanks. I just confirmed we are  doing the same for other Spanish translations.

Reply:
Yeah, I try to keep them in sync.